man/systemd.xml

```diff
@@ -74,8 +74,8 @@
     configuration files, whose syntax and basic set of options is
     described in
     <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-    however some are created automatically from other configuration,
-    dynamically from system state or programmatically at runtime.
+    however some are created automatically from other configuration
+	files, dynamically from system state or programmatically at runtime.
```

Comment:
please use spaces for indentation, not tabs

Reply:
Sorry about that, I didn't read the style guidelines.